docs/rules/0131/request-has-name-field.md

```diff
@@ -1,11 +1,11 @@
 ---
 rule:
   aip: 131
-  name: [core, '0131', request-name-field]
+  name: [core, '0131', request-has-name-field]
```

Comment:
How about `request-required-fields` for the presence check, and `request-name-field` for the type check?
This will need to be done in AIPs 132 through 135 as well for consistency.

Reply:
Done. PTAL. Thank you!